components/ExperienceCard.tsx

```diff
@@ -0,0 +1,44 @@
+"use client"
+import React from 'react';
+import {Card, CardHeader, CardBody, Image} from "@nextui-org/react";
+
+type Props = {
+    experiences: { 
+        data: 
+        { 
+            img: string; 
+            employer: string; 
+            job_title: string; 
+            years_worked: string;
+        }[];
+    };
+}
+
+const ExperienceCard = (props: Props) => {
+  return (
+    <>
+        {props.experiences.data.map((exp, idx) => (
+            <Card key={idx} className="Card Card:hover w-[345px] py-[36px] pl-[36px] rounded-[10px] gap-[18px] sm:w-[607px] dark:bg-transparent hover:dark:bg-black-300">
```

Comment:
```suggestion
            <Card key={idx} className="Card w-[345px] py-[36px] pl-[36px] rounded-[10px] gap-[18px] sm:w-[607px] dark:bg-transparent hover:dark:bg-black-300">
```
You shouldn't need the card:hover here

It looks like you're still adding to this PR. Let me know if you're done with this feature now and want me to merge or not :)